src/cogs/owner.py

```diff
@@ -919,21 +956,7 @@ async def makedir(self, ctx: Context, name: str):
         """Makes a directory for sprites to go in."""
         os.mkdir(f'data/sprites/{name}')
         with open(f'data/custom/{name}.toml', mode='x') as f:
-            f.write("""# ----------------------------------------------------------------------------
-# 
-# 
-#     Tile format:
-#         baba = { sprite = "baba", color = [ 0, 3 ], tiling = 2 } 
-#     Please do not use multiline tables, as it will cause merge conflicts with Git. 
-# 
-#     Note: If your tile has tiling type 1,
-#           you must add the "diagonal = true/false" attribute to the tile
-#           in order for it to pass CI.
-#           For example, "line" = { ..., diagonal = false }. 
-# 
-# 
-# ----------------------------------------------------------------------------            
-""")
+            f.write("# Please read CONTRIBUTING.md for guidance on how to properly edit this file.\n\n\n")
```

Comment:
Also seems copy pasted?

Reply:
think so here too

Reply:
This might need to be a constant or something.

Reply:
i searched around and it seems this is the only occurrence of this string now so idk